docs/usage/configuration-options.md

```diff
@@ -2108,10 +2108,11 @@ Consider this example:
 
 With the above config, every PR raised by Renovate will have the label `dependencies` while PRs containing `eslint`-related packages will instead have the label `linting`.
 
-Renovate only adds labels when it creates the PR, which means:
+Behaviour details:
 
-- If you remove labels which Renovate added, it won't re-apply them
-- If you change your config, the new/changed labels are not applied to any open PRs
+- Renovate will add any configured labels to PRs at time of PR creation
+- If labels on an open Renovate PR are modified after PR creation (e.g. via a user manually, or any automated process) then Renovate won't "enforce" its configured labels on subsequent runs. i.e. it won't reset/reapply its configured `labels`
+- On GitHub, GitLab and Gitea: if a PR's labels weren't modified after PR creation, and there is a change to Renovate `labels` configuration for such a PR, then Renovate _will_ reset/reapply configured `labels` to align them with the new config
```

Comment:
This text is hard to understand. The problem is that we have multiple paths through the code depending on:

- if the user/something else changes the label
- when the label change happens
- on which platforms the label change happens

I think a picture or mermaid graph of the different "routes" would be great! We still need good text for those who can't see the picture or graph though.

Reply:
It seems okay to me. Not sure a mermaid graph would be needed for this as the logic is similar to how we deal with modified branches. Maybe we could reduce the amount of words here.

Something like:

```md
- Renovate automatically applies configured labels to PRs upon creation.
- If labels on an open Renovate PR are changed manually or through automation after creation, Renovate doesn't enforce its configured labels in subsequent runs.
- If labels remain unchanged after PR creation and Renovate's label configuration is modified, Renovate will update the labels to match the new configuration on GitHub, GitLab, and Gitea.
```